src/site/xdoc/building.xml

```diff
@@ -85,7 +85,7 @@ under the License.
       <p>
         To build Syncope documentation execute (from within the top-level source directory):
         <source>$ mvn -N -P site clean generate-resources</source>
-        The documentation artifacts are now be available under <tt>target/generated-docs/</tt>.
+        The documentation artifacts are now be available under <code>target/generated-docs/</code>.
```

Comment:
Are the changes under `src/site/xdoc` required? Not sure if they are going to badly affect the project website generation...
If ensure, please revert, thanks.